### DISCUSSION TITLE
Does the retryExchange apply to mutations?

### DISCUSSION THREAD

**Discussion by nandorojo** [urql-graphql/urql#3280] [Q&A]
I'd like to retry mutations if the backend returns a certain error code. Is this configurable with the `retryExchange`?

**Comment by kitten** [comment]
Yes, of course,

however, out of curiosity and no offense, what makes you think it wouldn't work with any operation?

**Reply by nandorojo** [reply]
My default assumption is that automatically retrying all mutations can be a bit risky. What if you buy something twice, etc. I suppose you can implement itempotency keys for all mutations to avoid this from the backend, but we don’t always do that. 

I figured the retry exchange only retried queries, since that’s a safer behavior. It didn’t mention mutations so that’s why I thought to ask.
